src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPEC_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        StringBuilder builderWordsForAllFile = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                builderWordsForAllFile.append(value).append(System.lineSeparator());
+                value = bufferedReader.readLine();
+            }
+
+            String[] wordsOfFile = builderWordsForAllFile.toString().toLowerCase()
```

Comment:
starting from this line, you can move your code our of `try` block - since the reading part is already over

Reply:
done